static/flow/views/program-editor-panel.js

```diff
@@ -164,6 +166,8 @@ var ProgramEditorPanel = function(options) {
         _this.autoSaveProgram(false, false);
 
         _this.lastChangeMoveBlock = false;
+
+        _this.updateAllBlocks();
```

Comment:
In the rest of this file `updateAllBlocks` is called after `autoSaveProgram` but you have added this after it.  Just checking if that order is important.

Reply:
I'll take a look.  Thanks!

Reply:
I changed it.  I couldn't break it, but it seems safer to follow the proper ordering.